### PR TITLE
doc/rados: link to pg setting commands

### PR DESCRIPTION
**Pull request by zdover23**
Link to the instructions for manually setting the number of PGs per pool, from the mention of placement groups. These instructions are included here in response to a request from Ronen Friedman on the occasion of the removal of links to the PGcalc (see https://github.com/ceph/ceph/pull/55899#pullrequestreview-1912940118).

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/rados/operations/pools.rst [hunk 1]
```diff
@@ -25,7 +25,10 @@ Pools provide:
   resources. When setting up multiple pools, set an appropriate number of PGs
   for each pool and for the cluster as a whole. Each PG belongs to a specific
   pool: when multiple pools use the same OSDs, make sure that the **sum** of PG
-  replicas per OSD is in the desired PG-per-OSD target range. 
+  replicas per OSD is in the desired PG-per-OSD target range. See :ref:`Setting
+  the Number of Placement Groups <setting the number of placement groups>` for
```
Comment (ronen-fr) [inline doc/rados/operations/pools.rst:29]: The link does not work for me.
And the only target I can see is 'setting number ...' (i.e. - without the 'the'

Comment (zdover23) [reply doc/rados/operations/pools.rst:29]: @ronen-fr:

I have no trouble with the link. Have a look at the rendered document and let me know if you can't follow the link:

https://ceph--55918.org.readthedocs.build/en/55918/rados/operations/pools/#pools

Comment (ronen-fr) [reply doc/rados/operations/pools.rst:29]: Thanks. Indeed it works.

Comment (zdover23) [reply doc/rados/operations/pools.rst:29]: Qapla.

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/55935 - Squid backport
https://github.com/ceph/ceph/pull/55936 - Reef backport
https://github.com/ceph/ceph/pull/55937 - Quincy backport

**Pull request merged**